README.md

```diff
@@ -68,6 +73,29 @@ after running `make help`. To build all output products use:
 make all
 ```
 
+# Prepping the SD card for ZCU106 board
+
+In case the board runs from SD card (as in the case of the ZCU106 board), we can run:
+```
+make sdcard
+```
+
+This will create a directory `build/<target>/sdcard` directory with all files needed to run the project on the target.
+
+To prepare the SD card for the system, plug it to the computer and determine its device handle (e.g. `/dev/sdX`, where `X` is the letter corresponding to the SD card).
+
+**NOTE**: It is crucial to correctly determine the device, since the SD card will be wiped before placing files.
+
+After this, run:
+
+```
+./scripts/format-sdcard.sh /dev/<devname>
+```
+
+And conduct the formatting process.
+
+After this, mount the FAT partition and copy the contents of the `build/<target>/sdcard` directory to this directory.
```

Comment:
It's easy to get the wrong idea about the final partitioning of the SD card. One may understand this description in the way that there is only one partition, while there are two: `boot` and `root`. Maybe we can add a hint that the `boot` partition formated as FAT is the smaller one.

Reply:
Updated information with the list of created partitions